recipes/numpy_indexed/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: numpy-indexed
```

Comment:
It seems that this has a `-` in some cases and an `_` in others. Please let me know what was intended here.